code/part-two/README.md

```diff
@@ -77,28 +77,28 @@ source code and some prepackaged Sawtooth components downloaded from
 | ------------- | --------------------- | --------- | ------------------------
 | validator     | tcp://localhost:4004  | DockerHub | Validates blocks and transactions
 | rest-api      | http://localhost:8008 | DockerHub | Provides blockchain via HTTP/JSON
-| processor     | --                    | custom    | The core smart contract logic for your app
-| client        | http://localhost:3000 | custom    | Your front-end, served from `client/public/`
+| processor     | --                    | custom    | Your transaction processor (the core smart contract logic for your app)
+| client        | http://localhost:3000 | custom    | Your front end, served from `client/public/`
 | shell         | --                    | DockerHub | Environment for running Sawtooth commands
 | settings-tp   | --                    | DockerHub | Built-in Sawtooth transaction processor
 
 ### Start/Stop Commands
 
-First you will use the `docker-compose up` command to start all components.
+First, you will use the `docker-compose up` command to start all components.
 Note that this might take as long as 30 minutes the first time you run it
-(later runs will be much faster, more like 30 seconds). If you are in the same
-directory as the `docker-compose.yaml` file, the `up` command will find it by
-default and you won't need to provide any other parameters:
+(later runs will be much faster; more like 30 seconds). If you are in the same
+directory as the `docker-compose.yaml` file, the `up` command will find the file
+by default, so you won't need to provide any other parameters:
 
 ```bash
 cd code/part-two/
 docker-compose up
 ```
 
-This builds and starts up _everything_ defined in your compose file. Once
-running, you can then stop it all with the keyboard shortcut: `ctrl-C`. If you
-want to stop _and_ destroy every container (they will be rebuilt on your next
-`up`) use the command:
+This builds and starts _everything_ defined in your compose file. Once
+everything is running, you can then stop it all with the keyboard shortcut
```

Comment:
Using everything feels redundant here. If "Once running" is too vague, perhaps we can just vary up the word choice or something. Perhaps "Once all the components are running".

Reply:
I like your suggestion of "Once all the components are running". 

P.S. It's not that "Once running, you ..." is too vague, it's that this is a classic dangling modifier. It means "Once you are running". Isn't English fun?  :-)